tests/e2e/test_client.py

```diff
@@ -236,6 +236,58 @@ def test_upload_csv_success(self):
             True,
         )
 
+    @unittest.skip("This is a plus subscription endpoint.")
+    def test_create_table_success(self):
+        # Make sure the table doesn't already exist.
+        # You will need to change the namespace to your own.
+        client = DuneClient(self.valid_api_key)
+
+        self.assertEqual(
+            client.create_table(
+                namespace="test",
+                table_name="dataset_e2e_test",
+                description="e2e test table",
+                schema=[
+                    {"name": "date", "type": "timestamp"},
+                    {"name": "dgs10", "type": "double"},
+                ],
+                is_private=False,
+            ),
+            None,
```

Comment:
What's the `None` here ? Can we test based on the HTTP status code ?

Reply:
The `None` is what is returned right now from the API, but we are changing that
https://linear.app/dune/issue/PRO-1040/return-table-name-and-namespace-in-table-creation-response 

I believe we lose the context on the `status_code` from the way `_post` is implemented.
I think that package needs some work in general, but I didn't want to touch too many things

Reply:
> I believe we lose the context on the status_code from the way _post is implemented.
I think that package needs some work in general, but I didn't want to touch too many things

So we throw away the status code. Interesting 😄  Not much we can do then without reworking the whole API.

When returning `None` is not even clear if it's success or not. Let's leave as is.

Reply:
That's how ingestrs does it for `insert` now 😅 . 
But theoretically we should be able to use the status code if we were using `requests` or `http` package directly. 

But the implementation of the `BaseDuneClient` changes the response to `response.json()`, so the context is lost
https://github.com/duneanalytics/dune-client/blob/e3f6c98eab115e4f03d256b18fc8709a524c886f/dune_client/api/base.py#L139-L150

It's too big of change to change that now, lmk if you disagree

Reply:
Let's leave as is.